apps/go-mod-validator/src/go-mod-validator.ts

```diff
@@ -42,18 +42,18 @@ export async function run(): Promise<string> {
   const { goModDir, gh, depPrefix } = getContext();
 
   const depsToValidate = await getDeps(goModDir, depPrefix);
-
-  const errs: Map<BaseGoModule, string> = new Map();
+  interface Invalidation {
+    type: "error" | "warning";
+    msg: string;
+  }
```

Comment:
Why inline the interface?

Reply:
We dont need to expose it anywhere yet

Reply:
Maybe this is personal preference but I'd rather see the type in-lined into the Map (as it's only used once), or the interface outside the method.

Thoughts?

Reply:
I figured that this interface is going to be expanded upon in the future during a refactoring, its a core data structure. I'd rather keep it as is